### ISSUE TITLE
ignore_lines keys are not stringified from YAML

### ISSUE THREAD

**Issue by markedmondson** [glebm/i18n-tasks#206]
When using the YML format for scanner configuration like:

```
  scanners:
  - - "::I18n::Tasks::Scanners::PatternWithScopeScanner"
    - exclude:
      - "*.rb"
      ignore_lines:
        opal: ^\s*#(?!\si18n-tasks-use)
        haml: ^\s*-\\s*#(?!\si18n-tasks-use)
        coffee: ^\s*#(?!\si18n-tasks-use)
```

The `ignore_lines` keys are not stringified, therefore when they hit the [pattern scanner](https://github.com/glebm/i18n-tasks/blob/e8db4a195eefd3005f46d2aea6595a5671be8771/lib/i18n/tasks/scanners/pattern_scanner.rb#L51), they're not found.

The fix is as simple as: https://github.com/markedmondson/i18n-tasks/commit/17925c485e5835734d623a45de5f6e4ec5f97ccf

**Comment by glebm** [comment]
The keys in the example you've given are already Strings!

```console
> require 'yaml'; YAML.load 'x: 1'
 => {"x"=>1}
```

**Comment by markedmondson** [comment]
I thought the same thing so had to double check. A test case could probably be written around this but here's an awkward gif to show it.

![i18n](https://cloud.githubusercontent.com/assets/1653699/19898643/20a6411e-a01a-11e6-8a9a-4ccbbdbc675e.gif)

**Comment by glebm** [comment]
Ah, perhaps it's because of this:

https://github.com/glebm/i18n-tasks/blob/e8db4a195eefd3005f46d2aea6595a5671be8771/lib/i18n/tasks/used_keys.rb#L88

**Comment by glebm** [comment]
Your current solution (`ext.to_s`) seems good to me, please send a PR! Thanks

**Comment by glebm** [comment]
Actually, scratch that, I'll fix it in a way that works for scanners other than pattern scanners

**Comment by glebm** [comment]
Ah, other scanners don't have that setting

**Comment by markedmondson** [comment]
Great - thanks for the prompt response!

**Comment by glebm** [comment]
Released v0.9.6!
